src/main/java/com/thealgorithms/strings/Pangram.java

```diff
@@ -22,19 +23,12 @@ public static void main(String[] args) {
      * @return {@code true} if s is a Pangram, otherwise {@code false}
      */
     public static boolean isPangram(String s) {
-        boolean[] lettersExisting = new boolean[26];
-        for (char c : s.toCharArray()) {
-            int letterIndex = c - (Character.isUpperCase(c) ? 'A' : 'a');
-            if (letterIndex >= 0 && letterIndex < lettersExisting.length) {
-                lettersExisting[letterIndex] = true;
-            }
-        }
-        for (boolean letterFlag : lettersExisting) {
-            if (!letterFlag) {
-                return false;
-            }
-        }
-        return true;
+        // an approach using Java Collection Framework
+        Set<Character> alpha = new HashSet<Character>();
+        for (int i=0; i<s.length(); i++)
+            if (s.charAt(i)!=' ') alpha.add(s.charAt(i));
+        if (alpha.size() == 26) return true;
+        return false;
```

Comment:
This approach will not work as 'a' and "A" is different.
for example 'The quick brown fox jumps over the lazy dog' will return false as it will count T and t twice and the size will be greater than 26.
It is better if you change the char to lower case and apply this.

And please leave the previous algorithm there as it is more simple and clear and make another function with this alternate approach.

Reply:
Well I get that. But certainly if the string is converted to Lower Case and this approach is taken into count, it should be working fine. Isn't it?

Reply:
Also I will update this Java Collections approach in another function separately.

Reply:
I have committed the fresh changes in another branch and not in this one